### PR TITLE
net-p2p/freenet: add 0.7.5_p1497 | dev-java/freenet-ext: new package, add 29

### PR DESCRIPTION
**Pull request by vaukai**

### CODE REVIEW

File: licenses/BSD-3-Clause [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2013, Mitchell Bösecke
```
Comment (thesamesam) [inline licenses/BSD-3-Clause:1]: How is this different from BSD?

Comment (vaukai) [reply licenses/BSD-3-Clause:1]: > How is this different from BSD?

java-ebuilder finds what's given in  https://github.com/PebbleTemplates/pebble/blob/3.1.6/pom.xml#L28. Missing `BSD-3-Clause` urges packager to search and adjust manually in the ebuild.

Comment (thesamesam) [reply licenses/BSD-3-Clause]: Looks like maybe java-ebuilder needs to gain support for `metadata/license-mapping.conf`?

Comment (vaukai) [reply licenses/BSD-3-Clause]: > Looks like maybe java-ebuilder needs to gain support for `metadata/license-mapping.conf`?

https://bugs.gentoo.org/907208

**Pull request closed without merging**